go-controller/pkg/cni/helper_linux.go

```diff
@@ -3,23 +3,21 @@
 package cni
 
 import (
-        "fmt"
+	"fmt"
 	"io/ioutil"
 	"net"
 	"os"
 	"os/exec"
 	"strconv"
 	"strings"
 
-	"k8s.io/klog"
-
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
-
-	"github.com/Mellanox/sriovnet"
 	"github.com/containernetworking/cni/pkg/types/current"
 	"github.com/containernetworking/plugins/pkg/ip"
 	"github.com/containernetworking/plugins/pkg/ns"
 	"github.com/vishvananda/netlink"
+	"k8s.io/klog"
```

Comment:
I think that everything is in `"k8s.io/klog/v2"` now, is it possible you need to rebase or is a leftover?

Reply:
definitely needs rebase against master! however ATM this is based on the project's `bluefield` branch

Reply:
yeah, something like that, IIRC , if you use both at same time one of those doesn´t log anything